config/locales/translations/zendesk_apps_support.yml

```diff
@@ -204,3 +204,19 @@ parts:
       key: "txt.apps.admin.error.app_build.invalid_type_parameter.other"
       title: "App builder job: multiple invalid parameter types error"
       value: "%{invalid_types} are invalid parameter types."
+  - translation:
+      key: "txt.apps.admin.error.app_build.unacceptable_boolean"
+      title: "App builder job: this value needs to be either true or false, but something else was passed in"
+      value: "%{field} must be a Boolean value, got %{value}"
+  - translation:
+      key: "txt.apps.admin.error.app_build.invalid_no_template"
+      title: "App builder job: `noTemplate` needs to be true, false or a valid array, but something else was passed in"
+      value: "noTemplate must be set to true, false, or an array of valid locations."
```

Comment:
does noTemplate need to stay as is? If so, would you add this info in the title field? What about true and false? I guess they need to be in English too as they are code references? Please let me know